cli/command/job.go

```diff
@@ -10,6 +13,76 @@ func NewJobCommand(b *CommandBase) *JobCommand {
 	return &JobCommand{b}
 }
 
-func (e *JobCommand) Command() cli.Command {
-	return cli.Command{}
+func (j *JobCommand) Command() cli.Command {
+	return cli.Command{
+		Name:  "job",
+		Usage: "manage layer0 jobs",
+		Subcommands: []cli.Command{
+			{
+				Name:      "delete",
+				Usage:     "delete a job",
+				Action:    j.delete,
+				ArgsUsage: "NAME",
+			},
+			{
+				Name:      "read",
+				Usage:     "describe a job",
+				Action:    j.read,
+				ArgsUsage: "NAME",
+			},
+			{
+				Name:      "list",
+				Usage:     "list all jobs",
+				Action:    j.list,
+				ArgsUsage: " ",
```

Comment:
[nit] Should we omit this line?

Reply:
The `cli` package we use will throw up some custom help text for positional arguments if this field is an empty string - we just set it as a space here to stop it from doing that behavior.